docs/hacker-stats-intro/data.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
```

Comment:
I think the entire `data.py` can be placed inside the project's `src/bayes_tutorial/data.py`; there's some data loaders already in there. Inside the notebooks, it should be a small change to get the data imported:

```python
from bayes_tutorial.data import <the necessary functions>
```

Reply:
HBA to move all data imports into `src/bayes_tutorial/data.py` (w/out duplication).

Edit import calls in NBs